[2023-12-04 21:25:12,364 - INFO - main] - Test logging
[2023-12-04 22:34:03,816 - INFO - main] - Error occured in python script [D:\projects\Predictive_Maintenance\main.py] at line number [9] and message is [division by zero]
[2023-12-04 23:11:56,457 - INFO - common] - Connecting to database
[2023-12-04 23:11:57,918 - INFO - common] - connection established with database
[2023-12-04 23:11:57,918 - INFO - main] - Connection established
[2023-12-09 14:39:40,842 - INFO - common] - Error occured in python script [D:\projects\Predictive_Maintenance\src\predictive_maintenance\utils\common.py] at line number [20] and message is [SSL handshake failed: ac-yivjwqq-shard-00-01.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms),SSL handshake failed: ac-yivjwqq-shard-00-00.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms),SSL handshake failed: ac-yivjwqq-shard-00-02.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 65742eb683958e2c859a95c6, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('ac-yivjwqq-shard-00-00.nyvwsjt.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-yivjwqq-shard-00-00.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>, <ServerDescription ('ac-yivjwqq-shard-00-01.nyvwsjt.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-yivjwqq-shard-00-01.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>, <ServerDescription ('ac-yivjwqq-shard-00-02.nyvwsjt.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-yivjwqq-shard-00-02.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>]
[2023-12-09 14:42:57,129 - INFO - common] - Error occured in python script [D:\projects\Predictive_Maintenance\src\predictive_maintenance\utils\common.py] at line number [18] and message is [SSL handshake failed: ac-yivjwqq-shard-00-02.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms),SSL handshake failed: ac-yivjwqq-shard-00-00.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms),SSL handshake failed: ac-yivjwqq-shard-00-01.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 65742f7a4316ab524b9ddf22, topology_type: ReplicaSetNoPrimary, servers: [<ServerDescription ('ac-yivjwqq-shard-00-00.nyvwsjt.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-yivjwqq-shard-00-00.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>, <ServerDescription ('ac-yivjwqq-shard-00-01.nyvwsjt.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-yivjwqq-shard-00-01.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>, <ServerDescription ('ac-yivjwqq-shard-00-02.nyvwsjt.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('SSL handshake failed: ac-yivjwqq-shard-00-02.nyvwsjt.mongodb.net:27017: [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:1122) (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>]
